experimental/fairseq_moe_docker_slurm/ndv4-topo.xml

```diff
@@ -0,0 +1,34 @@
+<system version="1">
```

Comment:
This topology file is contained on the ubuntu-hpc image, why not pass that mount point to the container instead of including this file here?

Reply:
My first impression seeing this file in the launch.sh script was to locate the file on the login node but couldn't find it. I later find it on the compute nodes. 

I guess we could either provide it here or explicitly ask user to bind /opt:/workspace/opt in the job script.